persist-cli/src/main/java/io/ballerina/persist/cmd/Migrate.java

```diff
@@ -865,6 +970,228 @@ private static void convertFKMapToQuery(QueryTypes type, Map<String, List<Foreig
+                    "User needs to manually remove the column from the " + entry.getKey() + " table.";
+                    warnings.add(warning);
+                }
+                break;
+
+            case REMOVE_FOREIGN_KEY:
+                for (Map.Entry<String, List<String>> entry : map.entrySet()) {
+                    String entity = entry.getKey();
+                    for (String field : entry.getValue()) {
+                        String[] fieldData = field.split(",");
+                        String foreignKeyName = fieldData[0];
+                        warning = "WARNING: User must manually delete the foreign key " + 
+                        foreignKeyName + " in the " + entity + " table.";
+                        warnings.add(warning);
+                    }
+                }
+                break;
+
+            default:
+                break;
+        }
+
+        return warnings;
+    }
+
+    // Convert map of ForeignKey lists to a Google Sheets query
+    private static List<String> convertFKMapToGSheetsQuery(QueryTypes type, Map<String, List<ForeignKey>> map, 
+        List<String> queries) {
+        List<String> warnings = new ArrayList<>();
+        String warning = "";
+        if (Objects.requireNonNull(type) == QueryTypes.ADD_FOREIGN_KEY) {
+            for (Map.Entry<String, List<ForeignKey>> entry : map.entrySet()) {
+                String entity = entry.getKey();
+                for (ForeignKey foreignKey : entry.getValue()) {
+                    String childColumnName = foreignKey.getColumnName();
+                
+                    StringBuilder query = new StringBuilder(
+                        System.lineSeparator() + "  yourNewSheet  = activeSpreadsheet.getSheetByName(\"" + 
+                        entity + "\");" +
+                        System.lineSeparator() + "  var numColumns = yourSheet.getLastColumn();" +
+                        System.lineSeparator() + "  var newColumn = numColumns + 1;" +
+                        System.lineSeparator() + "  yourSheet.getRange(1, newColumn).setValue(\"" + 
+                        childColumnName + "\");"
+                    );
+
+                    query.append("\n);\n");
+
+                    queries.add(query.toString());
+
+                    warning = "WARNING: When deleting parent table user must manually remove any " +
+                    "foreign keys related to the deleted table.";
+                    warnings.add(warning);
+                }
+            }
+        }
+
+        return warnings;
+    }
+
+    private static String checkDataStore(String dataStore) {
+        switch (dataStore) {
+            case "mysql":
+                return ".sql";
+    
+            case "googlesheets":
+                return ".gs";
+    
+            default:
+                errStream.println("Error: Invalid data store: " + dataStore);
+                return "";
```

Comment:
Shall we throw the error instead of printing it as this is used in some places like line 386